packages/indexers/src/interfaces/IEVMIndexer.ts

```diff
@@ -32,6 +32,8 @@ export interface IEVMIndexer extends IIndexer {
     accountAddress: EVMAccountAddress,
     startTime: Date,
     endTime?: Date,
+    startBlockNumber?: number,
```

Comment:
Were these just missing from the interface? They aren't referenced anywhere else in the PR.